### PR TITLE
Bugfix and typofix

### PR DESCRIPTION
**Pull request by dmos62**
Two omitted breaks in switch statements and one typo. One of the omitted breaks is serious-ish, info in commit messages.

### CODE REVIEW

**Review by sqrrm — APPROVED** [review]
utACK

**Comment by sqrrm** [comment]
@ManfredKarrer I merged too soon. I didn't look at the two commits touching the DAO until after merging. I think it's correct but sensitive stuff.

File: core/src/main/java/bisq/core/dao/governance/proposal/param/ChangeParamValidator.java [hunk 1]
```diff
@@ -144,6 +144,7 @@ private void validateBsqValue(Coin currentParamValueAsCoin, Coin inputValueAsCoi
             case REIMBURSEMENT_MIN_AMOUNT:
                 checkArgument(inputValueAsCoin.value >= Restrictions.getMinNonDustOutput().value,
                         Res.get("validation.amountBelowDust", Restrictions.getMinNonDustOutput().value));
+                break;
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/governance/proposal/param/ChangeParamValidator.java:147]: I think the correct version would be:
```
            case REIMBURSEMENT_MIN_AMOUNT:
            case COMPENSATION_REQUEST_MAX_AMOUNT:
            case REIMBURSEMENT_MAX_AMOUNT:
                checkArgument(inputValueAsCoin.value >= Restrictions.getMinNonDustOutput().value,
                        Res.get("validation.amountBelowDust", Restrictions.getMinNonDustOutput().value));
                checkArgument(inputValueAsCoin.value <= 200000000,
                        Res.get("validation.inputTooLarge", "200 000 BSQ"));
                break;
```
Also MinAmount must not exceed the 200 000 BSQ.

File: core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java [hunk 2]
```diff
@@ -285,14 +285,15 @@ private boolean isBtcOutputOfBurnFeeTx(TempTxOutput tempTxOutput) {
                         // We always have the BSQ change before the burned BSQ output if both are present.
                         checkArgument(optionalOpReturnIndex.isPresent());
                         if (index != optionalOpReturnIndex.get() - 1) {
-                            return false;
+                            break;
```
Comment (ManfredKarrer) [inline core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java:288]: I don't see any problem with that change but as it is a pure "cosmetic" change I prefer to not touch the code. Some could argue the early return is more readable. I don't have a strong opinion and the break is more consistent but evaluating benefit / risk just does not justify the change IMO even it seems a trivial and very low risk change.

**Review by ManfredKarrer — COMMENTED** [review]
NACK

File: core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java [hunk 3]
```diff
@@ -285,14 +285,15 @@ private boolean isBtcOutputOfBurnFeeTx(TempTxOutput tempTxOutput) {
...
                         }
 
                         // Without checking the fee we would not be able to distinguish between 2 structurally same transactions, one
                         // where the output is burned BSQ and one where it is a BSQ change output.
                         long blindVoteFee = daoStateService.getParamValueAsCoin(Param.BLIND_VOTE_FEE, tempTxOutput.getBlockHeight()).value;
                         return availableInputValue == blindVoteFee;
                     }
+                    break;
```
Comment (dmos62) [inline core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java:296]: If the semantic changes in this file are reverted, take care not to revert adding this `break`.

Comment (sqrrm) [reply core/src/main/java/bisq/core/dao/node/parser/TxOutputParser.java:296]: I'm doing a revert of the whole PR. There was an easy to use button to do the revert.

Best make a new PR with the good changes.

**Pull request merged**